<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modConvert</class>
 <widget class="QMainWindow" name="modConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniMOD Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,0,4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Title: n/a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="origSizeLabel">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Original MOD size: n/a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="miniSizeLabel">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>New miniMOD size: n/a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Original MOD</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="sampleSelect">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sampleLenLabel">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Length:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sampleVolLabel">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Volume:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sampleFineLabel">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Repeats:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>New miniMOD</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,2,2">
         <item>
          <layout class="QGridLayout" name="gridLayout" columnstretch="5,3" columnminimumwidth="5,3">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Instrument</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="newTypeBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Volume</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="volumeBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Pitch Change</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Diminish</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="diminishBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="offsetBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>-50</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="triButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Triangle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sawButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Sawtooth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="squareButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Square</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="noiseButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Noise</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="songPlayButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play miniMod</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopSongButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="origPlayButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play Sample</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="miniPlayButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play miniType</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="lineGraph" name="graph" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open .mod</string>
   </property>
   <property name="iconText">
    <string>Open .mod</string>
   </property>
   <property name="toolTip">
    <string>Open .mod</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save  .c</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>lineGraph</class>
   <extends>QWidget</extends>
   <header location="global">linegraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
